build.gradle

```diff
@@ -116,15 +116,36 @@ repositories {
     maven {
         url "http://tehnut.info/maven"
     }
+	
+	// url of the maven that hosts Enchiridions files
+	maven {
+		url "http://girafi.dk/maven/"
+	}
+	
+	maven {
+		name = "Jared"
+		url = "http://maven.blamejared.com/"
+	}
```

Comment:
These lines are indented using the tab character. Can you change them to be 4 spaces instead?